store/src/qty.rs

```diff
@@ -507,22 +507,17 @@ impl Qty {
     true
   }
 
-  pub fn is_empty(&self) -> bool {
+  pub fn is_zero(&self) -> bool {
     if self.inner.is_empty() {
-      true
+      return true;
```

Comment:
why you need `return`?

Reply:
Error without return: expected `()`, found `bool`